src/passes/literalExpressionEvaluator/literalExpressionEvaluator.ts

```diff
@@ -99,7 +99,7 @@ function evaluateLiteral(node: Literal): RationalLiteral | boolean | null {
 
 function evaluateUnaryLiteral(node: UnaryOperation): RationalLiteral | boolean | null {
   const op = evaluateLiteralExpression(node.vSubExpression);
-  if (op === null) return null;
+  if (!op) return null;
```

Comment:
Nitpick: could you revert this change back. I believe that by explicitly stating that you want it to be different than null makes the code safer and more readable